app/helpers/application_helper.rb

```diff
@@ -550,4 +550,16 @@ def first_paragraph(full_text, placeholder_text = 'No preview available.')
       return content_tag(:p, ts(placeholder_text), class: 'placeholder')
     end
   end
+
+  # change the default link renderer for will_paginate
+  def will_paginate(collection_or_options = nil, options = {})
```

Comment:
Let's find out! (It's been broken on my development environment since one of the 4.x upgrades, so I can't tell by testing there.)